INSTRUCTION.md

```diff
@@ -0,0 +1,17 @@
+# Інструкції з запуску та зупинки контейнерів
+
+### Запуск контейнерів:
+1. Використовуйте команду:
+   docker-compose up
+Додаток буде доступний за адресою: http://localhost:8000/
```

Comment:
The instructions mention that the application will be available at http://localhost:8000, but the Dockerfile exposes port 8080. Ensure that the port in the instructions matches the port exposed in the Dockerfile to avoid confusion. You may need to update either the instructions or the Dockerfile accordingly.